CONTRIBUTING.md

```diff
@@ -92,28 +92,60 @@ It is possible to pass optional command line arguments to pytest, for example to
 run only certain tests using patterns (`-k`) or marker (`-m`).
 
 ```shell
-tox -e base -- <optional arguments>
+tox -e tests -- <optional arguments>
 ```
 
-Two important arguments are `--memcached-service` which allows to change the
-default of `localhost:11211` (memcached's default) to a different address, and
-`-n` which sets the number of parallel workers for pytest-xdist. There are two
-layers of parallelization in the tests. An inner one within the tests
-themselves, i.e. the parallelism in the algorithms, and an outer one by
-pytest-xdist. The latter is controlled by the `-n` argument. If you experience
-segmentation faults with the tests, try running them with `-n 0` to disable
-parallelization.
+There are a few important arguments:
 
-To test modules that rely on PyTorch, use:
+- `--memcached-service` allows to change the default of `localhost:11211` (memcached's default)
+  to a different address.
 
-```shell
-tox -e torch
-```
+  [Memcached](https://www.memcached.org/) is needed for testing
+  caching as well as speeding certain methods (e.g. Permutation Shapley).
+
+  To start memcached locally in the background with Docker use:
+
+  ```shell
+   run --name pydvl-memcache -p 11211:11211 -d memcached
+  ```
+
+- `-n` sets the number of parallel workers for 
+  [pytest-xdist](https://pytest-xdist.readthedocs.io/en/latest/).
+  
+  There are two layers of parallelization in the tests.
+  An inner one within the tests themselves, i.e. the parallelism in the algorithms,
+  and an outer one by pytest-xdist. The latter is controlled by the `-n` argument.
+  If you experience segmentation faults with the tests,
+  try running them with `-n 0` to disable parallelization.
+
+- `--slow-tests` enables running slow tests.
```

Comment:
I think this merits some explanation. What are "slow tests", why do we have such a category, when are they run by default (they always run on CI, correct?), etc.

Reply:
I updated CONTRIBUTING.md with more detail.

Things aren't really faster with xdist because of the configuration we are using. For me xdist only starts a single worker so it doesn't change anything.

Reply:
Maybe you don't have that many cores? It should instantiate (n_cpus-1) / 4 workers. The 4 comes from the inner parallelization of the algorithms, which we may want to reduce to 3 or even 2 if that allows starting more xdist workers. I have 32 cores so I am happily running very quick tests 😇

Reply:
Yes, I saw that and understood why that was happening. We can probably tune the values later if it turns out we need another speedup.

Reply:
Hmm... let me see if I we see things the same way:
* xdist is used locally, max number of cores, divided by 4. Probably could pick less than 4 processes for inner parallelization
* split is used on CI only, or locally, manually